tortoise/util.go

```diff
@@ -2,50 +2,61 @@ package tortoise
 
 import (
 	"fmt"
+
 	"github.com/spacemeshos/go-spacemesh/common/types"
 	"github.com/spacemeshos/go-spacemesh/log"
 )
 
-type vec [2]int
+// NOTE(dshulyak) there is a bug in xdr. without specifying size xdr will cut the value
+// down to int32
+type vec struct {
+	Support, Against int64
```

Comment:
We use uint64 for this in healing branch. see #2629

Reply:
can it be limited to uint32? will save some inuse memory. given that we will be storing millions of votes in mainnet

Reply:
based on last observation, opinion layers 18-1057. total 6173753. evict from 28. last 1057. verified 38. reducing it to uint32 will save ~50mb. so maybe won't be that important

Reply:
no, it needs to encode voting weights, and voting weights are uint64. this would be a very deep change to varpost.